.github/workflows/deploy-dev-gcp.yaml

```diff
@@ -1,10 +1,6 @@
 name: Deploy to dev
 on:
     workflow_dispatch:
-    push:
```

Comment:
Er det ikke fint å holde dev-miljøet oppdatert med siste endringer?

Reply:
Hmm, husker vi snakka om dette da vi gikk vekk fra dev-branch. Falt ned på at det ikke var nødvendig med automatisk deploy siden: Man tar vel som regel alltid og tester i dev før man prodsetter noe og da er det jo synka!
Siden man gjør det, så blir det bare unødvendig ressursbruk å kjøre enda en gang, og i verste kan man ufrivillig pushe til dev mens noen andre holder på der, hvis man prodsetter noe mens andre tester, og da må man kjøre enda en deploy!

Men så kan vel det motsatte oppstå også ja, at noe blir prodsatt uten test, og at det da havner ut av synk, men magefølelsen sier det er sjelden 🤔

Reply:
Det er forsåvidt et godt poeng, men jeg tror sannsynligheten for at branchen du legger ut for å teste i dev er "stale" er ganske høy likevel. Ser for meg følgende scenario:

* Jeg sjekker ut master med siste endringer
* Jeg brancher ut og lager en ny "work"-branch
* Jeg jobber på work-branchen
* _Time goes by_
* _Person 1 deployer noen andre endringer i mellomtiden_
* _Person 2 deployer noen andre endringer i mellomtiden_
* Mine endringer er klare og jeg deployer min - nå utdaterte branch - i dev

Reply:
Ja, det vil jo uansett skje når flere jobber på samme repo i forskjellig tempo, men det vil ordnes automatisk hvis man har automatisk deploy av master. Handler vel litt om filosofien rundt dev-miljøet, om det skal være et sted man i hovedsak deployer noe for å teste eller om det skal knyttes tett opp til prod-miljøet. 

Akkurat i porteføljeflate så ser jeg for meg at det er større sannsynlighet for at man ødelegger for noen som tester hvis man har automatisk deploy enn i de andre repoene våre 🤔